config/zephyr/Kconfig

```diff
@@ -267,6 +267,7 @@ config CHIP_OPERATIONAL_TIME_SAVE_INTERVAL
 
 config CHIP_CRYPTO_PSA
 	bool "Use PSA crypto API for cryptographic operations"
+	select EXPERIMENTAL
```

Comment:
Remember it's a common Zephyr module. I'm not sure we should select EXPERIMENTAL for all platforms. Maybe we should create a dummy nRF only kconfig that depends on CHIP_CRYPTO_PSA and selects EXPERIMENTAL.

Reply:
Oh, right. But we have assigned it as experimental in another config related to PSA, so it can be removed from there.